File: documentation-examples-java/src/java/dev/typr/foundations/docs/mariadb/VectorType.java
package dev.typr.foundations.docs.mariadb;

import dev.typr.foundations.MariaType;
import dev.typr.foundations.MariaTypes;
import dev.typr.foundations.data.Vector;

@SuppressWarnings("unused")
public class VectorType {
  // start
  MariaType<Vector> embedding = MariaTypes.vector(1536);
  // stop
}

File: documentation-examples-kotlin/src/kotlin/dev/typr/foundationskt/docs/mariadb/VectorType.kt
package dev.typr.foundationskt.docs.mariadb

import dev.typr.foundationskt.MariaType
import dev.typr.foundationskt.MariaTypes
import dev.typr.foundations.data.Vector

@Suppress("unused")
class VectorType {
    //start
    val embedding: MariaType<Vector> = MariaTypes.vector(1536)
    //stop
}
